design.html

```diff
@@ -0,0 +1,1145 @@
+                    <figcaption class="st-row-text2 border border-top-0 pb-3  text-justify " ><a class=" st-row-text font-weight-bold text-body" href="">Biden, the Moderate, Plans the Most Radical Economic Overhaul Since FDR</a><br>
+                        To revive the economy, Joe Biden wants to build his own New Deal with an array of proposals to
+                        strengthen the social safety net and revive prosperity. The big question: Is he up to the
+                        challenge?</figcaption>
+                </figure>
+                <figure >
+                    <img class="st-row-imgs sm-topstory-img2 w-100" src="./assets/things/newsweekapm.PNG" alt="politics">
+                    <figcaption class="st-row-text2 pb-3 border border-top-0 text-justify border-bottom" ><a class=" st-row-text font-weight-bold text-body" href="">Biden, the Moderate, Plans the Most Radical Economic Overhaul Since FDR</a><br>
+                        To revive the economy, Joe Biden wants to build his own New Deal with an array of proposals to
+                        strengthen the social safety net and revive prosperity. The big question: Is he up to the
+                        challenge?</figcaption>
+                </figure>
+                <figure >
+                  <img class="st-row-imgs sm-topstory-img2 w-100" src="./assets/things/img9row1.PNG" alt="politics">
+                  <figcaption class="st-row-text2 pb-3  text-justify border-bottom" ><a class=" st-row-text font-weight-bold text-body" href="">Biden, the Moderate, Plans the Most Radical Economic Overhaul Since FDR</a><br>
+                      To revive the economy, Joe Biden wants to build his own New Deal with an array of proposals to
+                      strengthen the social safety net and revive prosperity. The big question: Is he up to the
+                      challenge?</figcaption>
+              </figure>
+            </div>
+          <!----><div class="sm-topstory " >  <!---->
+                        <span > MORE STORIES</span>
+      
+               <div class="container  py-3 border-bottom ">
+                        <div class="media  text-justify col-md-3">
+                          <img src="./assets/things/middle8.PNG" class="align-self-center mr-3 "  alt="John Doe" >
+                          <div class="media-body">
+                            <a class=" h5 font-weight-bold text-body" href="">Ahmaud Arbery's Mother Says George Floyd's Death 'Sent Me Back'</a><br>
+                            <span class="h6">To revive the economy, Joe Biden wants to build his own New Deal with an array of proposals to strengthen the social </span>      
+                          </div>
+                        </div>
+                      </div>
+                      <div class="container py-3 border-bottom">
+                        <div class="media  text-justify">
```

Comment:
- [ ] The text needs to wrap below the image when the content is big enough.

| Original | Yours |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88103485-77391d80-cb77-11ea-997d-cb9234ae0396.png) | ![image](https://user-images.githubusercontent.com/61479648/88103545-8a4bed80-cb77-11ea-9739-dd98a735ddab.png)  |

- [ ] The line-height and font-size for the section 'My Turn' (in your project is 'Heroes of the Pandemic and 'Cultural and Travel') is wrong.


| Original | Yours |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88103742-d139e300-cb77-11ea-8cbb-1fb6b9650ec5.png) | ![image](https://user-images.githubusercontent.com/61479648/88103823-e878d080-cb77-11ea-888b-2841fc788f6d.png) |